regulation-worker/internal/delete/api/api.go

```diff
@@ -39,6 +41,31 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destConfig map
 
 	bodySchema := mapJobToPayload(job, strings.ToLower(destName), destConfig)
 	pkgLogger.Debugf("payload: %#v", bodySchema)
+	pkgLogger.Debugf("Regulation Delete API called:\n")
+
+	var tokenStatusCode int
+	var accountSecretInfo *oauth.AuthResponse
+	// identifier to know if the destination supports OAuth
+	// TODO: "rudderAccountId" has to change to "rudderUserDeleteAccountId"
+	rudderUserDeleteAccountId, delAccountIdExists := destConfig["rudderAccountId"]
+	// TODO: This needs to be changed
+	isOauthEnabled := delAccountIdExists || destName == "GA"
```

Comment:
We should define a separate collection for OAuth destination names and use that to validate rather than hardcoding in the code.

Reply:
Maybe that's what the above TODO is for.

Reply:
Basically this is for testing purpose only. The TODO is to change the logic of checking oauth enabling

Reply:
The changes have been made, can you verify ?